packages/event-bridge/src/consumer.ts

```diff
@@ -0,0 +1,53 @@
+import { Consumer, ConsumerOptions } from 'sqs-consumer';
+import { IncomingPocketEvent } from './events';
+import { type Message } from '@aws-sdk/client-sqs';
+import { serverLogger } from '@pocket-tools/ts-logger';
+import { sqsPollerEventBridgeEvent } from './utils';
+
+export type PocketEventBridgeSQSConsumerOptions = Omit<
+  ConsumerOptions,
+  'handleMessage' | 'handleMessageBatch'
+> & {
+  /**
+   * An `async` function (or function that returns a `Promise`) to be called whenever
+   * a Pocket Event is received.
+   * @param event
+   */
+  handlePocketEvent(event: IncomingPocketEvent): Promise<void>;
+};
+
+export class PocketEventBridgeSQSConsumer extends Consumer {
+  constructor(options: PocketEventBridgeSQSConsumerOptions) {
+    super({
+      messageAttributeNames: ['All'],
+      ...options,
+      visibilityTimeout: 10000,
+      handleMessage: async (message) => {
+        return await options.handlePocketEvent(
+          PocketEventBridgeSQSConsumer.handleMessage(message),
+        );
+      },
+    });
+  }
+
+  private static handleMessage(message: Message): IncomingPocketEvent {
+    const pocketEvent = sqsPollerEventBridgeEvent(message);
```

Comment:
Can you please try/catch this so we can handle issues with JSON.parse underlying methods (e.g. JSON.parse on an undefined) -- in case we receive messages that don't have expected boilerplate attributes from flowing through SNS

Reply:
It will fail regardless in this case -- but just so we have more informative errors and better grouping